docs/introduction.rst

```diff
@@ -1,10 +1,22 @@
 Introduction to Flatpak
 =======================
 
-Flatpak is a framework for distributing desktop applications on Linux. It
-has been created by developers who have a long history of working on the
+Flatpak is a framework for distributing desktop applications across various Linux
+distributions. It has been created by developers who have a long history of working on the
 Linux desktop, and is run as an independent open source project.
 
+Terminology
+-----------
+
+- Flatpak: The Flatpak package manager.
+- Flatpak application: These are the applications you install via the ``flatpak`` command or via a
+  different UI like GNOME Software or KDE Discover.
+- Runtime: Also called platforms, these are a collection of basic utilities needed for a
+  Flatpak application to work.
+- BaseApp: Similar to a runtime, but for frameworks like Electron.
```

Comment:
Not sure it's technically accurate to describe base apps as being similar to runtimes.